### PR TITLE
Changed variable values to improve driving, added simple HUD, added a…

### PR DESCRIPTION
**Pull request by Dimmitsaras**
…nti-rolling mechanics. And a ramp.

Still have to play with the numbers to improve it. I also made it steer less at high speeds to simulate drag without affecting acceleration but its not that clean yet.

### CODE REVIEW

**Comment by RileyEv** [comment]
Perfect, will have a play with it tomorrow @Nenivar @Keywarn. Think the steering less at high speeds is a good idea. We could possibly model it as a curve rather than thresholds?

**Comment by Keywarn** [comment]
This looks good, there are a few unity convention things to take a look at. And yeah, a curve could be used here but for now this is perfect.

File: ProjectFiles/Assets/Scripts/Car/AntiRoll.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+using UnityEngine;
+
+public class AntiRoll : MonoBehaviour
+{
+    public WheelCollider leftWheel;
+    public WheelCollider rightWheel;
+    public GameObject car;
+    private Rigidbody rb;
+
+    [SerializeField] public float AntiSwayMultiplier = 5000.0f;
```
Comment (Keywarn) [inline ProjectFiles/Assets/Scripts/Car/AntiRoll.cs]: We don't need `[SerializeField` here as the variable is public (should it be public?)

File: ProjectFiles/Assets/Scripts/Car/DriveController.cs [hunk 2]
```diff
@@ -33,6 +44,23 @@ void FixedUpdate()
...
+        else if (v > 80)
+            return 7;
+        else if (v > 200)
+            return 10;
+        else if (v > 400)
+            return 20;
+        else
+            return 1;
+    }
+
```
Comment (Keywarn) [inline ProjectFiles/Assets/Scripts/Car/DriveController.cs:63]: This works for now but could be a curve in the future

**Review by Keywarn — APPROVED** [review]

**Pull request merged**